bucket/pelook.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.73",
+    "description": "Information tool for Windows EXE, DLL, driver and OBJ files (PE/COFF images)",
+    "homepage": "http://bytepointer.com/tools/index.htm#pelook",
+    "license": {
+        "identifier": "Freeware",
+        "url": "http://bytepointer.com/tools/index.htm"
+    },
+    "url": "http://bytepointer.com/download.php?name=pelook.zip#/dl.zip",
+    "hash": "12baccfb7db419c7533bcf495392e1baf9740c8b3bc558e496201c3c1a046cd7",
+    "bin": "pelook.exe",
+    "checkver": {
+        "url": "http://bytepointer.com/tools/pelook_changelist.htm",
+        "regex": "\\d{4}-\\d{2}-\\d{2}\\s+([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "http://bytepointer.com/download.php?name=pelook.zip#/dl.zip",
```

Comment:
Do we really need to rename the ZIP file?

Reply:
If we don't rename it, the extraction will fail due to wrong file name.

Reply:
Any idea why?

Is it because of the ?name= in the URL?

Reply:
> Is it because of the ?name= in the URL?

It is. think this needs to be fixed in *Scoop*'s main program.